benchmark/gem.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
+require_relative '../result'
+
 module Benchmark
   # Abstract base class for benchmarking an SDK Gem.
   # Implementors must define the `gem_name`, `client_klass`, and the
   # `operation_benchmarks` methods.
+  # rubocop:disable Metrics/ClassLength
```

Comment:
For class length, the rubocop default is pretty low IMO. I'd just change it to 200 globally and you won't need this one.